evergreen/add_fake_appid.js

```diff
@@ -0,0 +1,6 @@
+// Add a version to the schema change history store so that the drop optimization does not take place
```

Comment:
might be worth clarifying this "drop otimization" takes place when PBS is being terminated or an app with PBS enabled is being deleted so it's clearer where this comes into play